.github/actions/task-checklist/index.js

```diff
@@ -0,0 +1,63 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+// Detects tasks in the form:
+// - [ ] Task 1
+//
+// Also handles whitespace around any of the characters,
+// and the three different ways to make a list (dash, asterisk, plus)
+const INCOMPLETE_TASKS_REGEX = /^\s*[-*+]\s+\[ \]\s+(.*)/gm;
+
+// This regex finds regions which we should EXCLUDE from the task checklist action.
+//
+// - s flag required so that ".*?" matches over multiple lines
+// - "?" in ".*?" ensures that we match the smallest possible string
+// - note that m flag is not required.
+const DISABLE_COMMENT_REGEX =
+  /<!--\s*task-checklist-ignore-start\s*-->.*?<!--\s*task-checklist-ignore-end\s*-->/gs;
+
+const run = () => {
+  const body = github.context.payload.pull_request?.body;
+  if (!body) {
+    console.log('PR description empty, skipping this check.');
```

Comment:
Could be a `core.warning` or even `core.notice` (to annotate) if you care at all.

Reply:
ah looks like the `core` functions https://github.com/actions/toolkit/tree/main/packages/core#annotations have better integrations with GitHub's UI, good idea

Reply:
Yeah, I just don't exactly know what `core.info` does, the rest is helpful!

Reply:
![Screenshot 2024-07-16 at 14 19 57](https://github.com/user-attachments/assets/1bc5c47d-01e7-421d-83f2-d0bc0c12b647)
![Screenshot 2024-07-16 at 14 20 42](https://github.com/user-attachments/assets/ed0d556a-48f9-4647-bc2b-a0ea04a48671)

nifty